enterprise_scale/construction_sets/aks/flux.tf

```diff
@@ -0,0 +1,91 @@
+provider "flux" {}
+
+provider "kubectl" {
```

Comment:
I recently deprecate these kubeconfig values due to them being sensitive and us avoiding to store those on tfstate. #52 
Consider to use data source for aks to retrieve these kubeconfig data

Reply:
Flux.tf would fit better in addons folder